src/sass/components/_advertising.scss

```diff
@@ -11,6 +11,26 @@
       line-height: 36px;
       color: #fff;
     }
+    @media (max-width: 576px) {
```

Comment:
Mixed Media-queries/rwd in plain stylesheet.
Please put your queries in _mediaquersies.scss file.

Reply:
sugestion implemented